module/modmanager/manager.go

```diff
@@ -365,19 +365,14 @@ func (mgr *Manager) Reconfigure(ctx context.Context, conf config.Module) ([]reso
 	mgr.logger.CInfow(ctx, "New module process is running and responding to gRPC requests", "module",
 		mod.cfg.Name, "module address", mod.addr)
 
-	// add old module process' resources to new module; warn if new module cannot
-	// handle old resource and consider that resource orphaned.
 	var orphanedResourceNames []resource.Name
-	for name, res := range handledResources {
-		if _, err := mgr.addResource(ctx, res.conf, res.deps); err != nil {
-			mgr.logger.CWarnw(ctx, "Error while re-adding resource to module",
-				"resource", name, "module", conf.Name, "error", err)
-			orphanedResourceNames = append(orphanedResourceNames, name)
-		} else {
-			mgr.logger.CInfow(ctx, "Successfully re-added resource from module after module reconfiguration",
-				"module", mod.cfg.Name, "resource", name)
-		}
+	var orphanedResourceNameStrings []string
+	for name := range handledResources {
+		orphanedResourceNames = append(orphanedResourceNames, name)
+		orphanedResourceNameStrings = append(orphanedResourceNameStrings, name.String())
 	}
+	mgr.logger.CInfow(ctx, "Resources handled by reconfigured module will be re-added to new module process",
```

Comment:
I thought the logs here and in `Remove` would be helpful for us/module authors. Let me know if you think otherwise.